packages/web-components/stencil.config.ts

```diff
@@ -54,6 +54,7 @@ export const config: Config = {
     }
   ],
   testing: {
+    browserArgs: ['--no-sandbox', '--disable-setuid-sandbox'],
```

Comment:
This was added because of a CI failure discussed here: https://dsva.slack.com/archives/G01BJ3ESXL4/p1728583704700209?thread_ts=1728509612.263229&cid=G01BJ3ESXL4